frontend/src/App.js

```diff
@@ -1,9 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Registration } from 'components/Registration'
+import { Signin } from 'components/Signin'
+import { ContentPage } from 'components/ContentPage'
 
 export const App = () => {
+  const [showRegister, setShowRegister] = useState(false)
+  const [showForm, setShowForm] = useState(true)
+  const [showContentPage, setShowContentPage] = useState(false)
```

Comment:
Maybe since we save accessToken in the localStorage we could `setShowContentPage` based on that?